PWGGA/GammaConv/macros/AddTask_OmegaToPiZeroGamma_pp.C

```diff
@@ -163,20 +183,29 @@ void AddTask_OmegaToPiZeroGamma_pp(  Int_t     trainConfig                   = 1
   if (isMC > 0) isMCForOtherSettings = 1;
   //========= Add PID Reponse to ANALYSIS manager ====
   if(!(AliPIDResponse*)mgr->GetTask("PIDResponseTask")){
-    gROOT->LoadMacro("$ALICE_ROOT/ANALYSIS/macros/AddTaskPIDResponse.C");
-    AddTaskPIDResponse(isMCForOtherSettings);
+
+    #if !defined (__CINT__) || defined (__CLING__)
+      AliAnalysisTaskPIDResponse *pidRespTask=reinterpret_cast<AliAnalysisTaskPIDResponse*>(
+          gInterpreter->ExecuteMacro(Form("$ALICE_ROOT/ANALYSIS/macros/AddTaskPIDResponse.C (%i)",isMCForOtherSettings)));
+    #else
+      gROOT->LoadMacro("$ALICE_ROOT/ANALYSIS/macros/AddTaskPIDResponse.C");
+      AddTaskPIDResponse(isMCForOtherSettings);
+    #endif
+
+    // gROOT->LoadMacro("$ALICE_ROOT/ANALYSIS/macros/AddTaskPIDResponse.C");
+    // AliAnalysisTaskPIDResponse* taskPIDResponse =  AddTaskPIDResponse(isMCForOtherSettings);
   }
 
   Printf("here \n");
 
   //=========  Set Cutnumber for V0Reader ================================
-  TString cutnumberPhoton = "00000008400100001500000000";
+  TString cutnumberPhoton = "10000008400100001500000000";// 00000008400100001500000000
```

Comment:
similar to, for example AddTask_GammaCalo_pp.C please make the cutnumberPhoton an argument of the AddTask itself. So that it can be handled by the global variables on our train pages.